### PR TITLE
Add blog: Unfug Programming Saturday

### PR DESCRIPTION
**Pull request by matthiasbeyer**
@TheNeikos and @fkemmer and @neithernut could you guys proof-read this?

### CODE REVIEW

**Comment by neithernut** [comment]
What is a "regular unfug attendee"?

**Comment by matthiasbeyer** [comment]
Someone who attends our thursday sessions at least _sometimes_.

Maybe my wording sucks here, not sure...

**Comment by neithernut** [comment]
I'd just drop the entire "closed-door" policy and replace it with "no tutoring for people too stupid for googling".

**Comment by neithernut** [comment]
...or just "no tutoring"

**Comment by matthiasbeyer** [comment]
I like the "too stupid" part... will update.

File: content/blog/2016/Unfug-Programming-Saturday/article.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+For this semester, the Unfug got the permission to host a weekly event, called
+the "Unfug Hackt" (unfug programming).
+
+The attendees of the Unfug will meet each Saturday (from 14:00, open end) to
+work on their private programming projects and to collaborate, socialize and to
+have fun together while working with and learning programming languages.
+
+We will not provide tutoring for people which are too stupid to use google and
+we are neither a programming-learning class nor a homework group.
```
Comment (neithernut) [inline content/blog/2016/Unfug-Programming-Saturday/article.md]: Better English would be `... to use google. Neither are we ...`

Comment (neithernut) [inline content/blog/2016/Unfug-Programming-Saturday/article.md]: Drop the and and make it two sentences.

**Comment by neithernut** [comment]
Ack.

**Comment by matthiasbeyer** [comment]
Should I wait for one more review or merge this and deploy it to our site?

**Comment by neithernut** [comment]
As long as it's up before the freshmen's introduction event next week...

**Comment by TheNeikos** [comment]
> We want to work efficient and in a quiet and concentrating environment on private projects.

And here I thought the memes around German could be stopped slowly!

**Comment by matthiasbeyer** [comment]
:smile:

**Pull request merged**